### PR TITLE
Goku

### PR DESCRIPTION
**Pull request by gokul-khoros**
try out pull request

### CODE REVIEW

File: src/MainApplication.java [hunk 1]
```diff
@@ -3,5 +3,6 @@
 public class MainApplication {
         public static void main(String[] args) {
             System.out.println("bonjour");
+            System.out.println("hola");
```
Comment (ManasviniArul) [inline src/MainApplication.java]: holaa

File: src/NextApplication.java [hunk 2]
```diff
@@ -1,2 +1,7 @@
-package src;public class NextApplication {
+package src;
+
+public class NextApplication {
+ private void dummyClass(){
```
Comment (ManasviniArul) [inline src/NextApplication.java:4]: space between ) & {

Comment (gokul-khoros) [reply src/NextApplication.java:4]: not nits dude 
fyi: i dont know what is nits or linters

Comment (ManasviniArul) [reply src/NextApplication.java:4]: sathyama purila ena solra nu :/

**Review by ManasviniArul — COMMENTED** [review]
good work

**Pull request closed without merging**